test/common/utils/validations.unit.spec.js

```diff
@@ -195,16 +196,16 @@ describe('Validations unit tests', () => {
     it('should validate partial range', () => {
       const fromDate = {
         month: {
-          value: '2'
+          value: 2
```

Comment:
Just for consistency as there seemed to be places they were strings and places they were numbers. Perhaps they should all be sent as strings then?